manager/manager.go

```diff
@@ -144,8 +144,9 @@ func (rm *realManager) HousekeepModel() {
 
 func (rm *realManager) Housekeep() {
 	for {
-		start := rm.lastSync
-		end := start.Add(rm.resolution)
+		end := rm.lastSync.Add(rm.resolution)
+		// We need only one sample every housekeeping. Issue #638.
+		start := end.Add(-time.Second)
```

Comment:
So, I head from @piosz that cAdvisor/Kubelet doesn't generate a new sample if the new one is more-or-less equal to the the previous and the previous was taken in the last 15 sec or so. I'm not sure where and how this logic is exactly implemented.  It is also possible that I misunderstood something.